main.py

```diff
@@ -84,6 +84,7 @@ def add_schedule():
         }
 
         schedules_model.Name = request.json["name"]
+        schedules_model.DisplayName = request.json["displayname"] or request.json["name"]
```

Comment:
Dict access via this syntax will throw a KeyError if the key is not present. Try this instead `request.json.get("displayname",request.json.get("name"))`